pages/glossary/aria-attribute-set.md

```diff
@@ -0,0 +1,40 @@
+---
+title: ARIA state or property is set
+key: aria-attribute-set
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An ARIA [state][aria state] or [property][aria property] is <dfn>set</dfn> on an [HTML element][namespaced element] when it has a value. This may happen in three ways:
+
+- It is <dfn id="aria-attribute-set:explicit">explicitly set</dfn> if there is a corresponding `aria-*` HTML attribute on the element. If the [value][html attribute value] written in the HTML code is not valid for this ARIA [state][aria state] or [property][aria property] then the attribute is set but does not have an [attribute value][].
+
+  For example, `aria-label` is explicitly set on `<button aria-label="Next page">Next</button>`; and `aria-checked` is explicitly set on `<input type="checkbox" aria-checked="yes" />`, even though it does not have an [attribute value][].
```

Comment:
just wondering if it would be better to use a different example for `<input type="checkbox" aria-checked="yes" />` since these attributes generally shouldn't be used together and the value of `checked` will be `false` on the accessibility tree